src/Common/Notifications/Controller.php

```diff
@@ -111,7 +111,7 @@ public function register_ian() {
 	 */
 	public function is_ian_page() {
 		$screen  = get_current_screen();
-		$allowed = [ 'tribe_events', 'edit-tribe_events', 'tribe_events_page_tec-events-settings' ];
+		$allowed = [ 'tribe_events', 'edit-tribe_events', 'tribe_events_page_tec-events-settings', 'tickets_page_tec-tickets-settings' ];
```

Comment:
It would really be better if ET hooked into the filter below and added itself. Ditto for TEC pages (but that's another ticket)

Reply:
We should always try to minimize the amount of plugin names, etc we use in TCMN - we have a hook, use it and let the plugin itself handle the strings.